index.html

```diff
@@ -155,6 +156,31 @@ <h3 class="language-title"><i class="fa-solid fa-circle icon"></i> Skills</h3>
       </ul>
     </div>
   </section>
+
+  <!-- Contact form -->
+  <section class="contact-form">
+    <h4 class="contact-title">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h4>
+    <form action="https://formspree.io/f/xnqrnjll" method="post">
+      <input type="text" class="form-field form-item" maxlength="30" name="fullname" id="fullname"
+        placeholder="Full Name" required>
+      <input type="email" class="form-field form-item" name="email" id="email" placeholder="Email Address" required><br>
+      <textarea name="message" id="message" maxlength="500" class="form-item" cols="30" rows="10"
+        placeholder="Enter text here" required></textarea><br>
+      <input type="submit" class="submit form-item" value="Get in Touch">
+    </form>
+  </section>
+
+  <footer>
+    <ul class="social-links">
+      <li><a href="#"><i class="fa-solid fa-envelope"></i></a></li>
+      <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+      <li><a href="#"><i class="fab fa-facebook"></i></a></li>
```

Comment:
[REQUIRED] Oe of the aims of this project is to teach you how to follow client requirements. Kindly ensure that your design matches the template as pointed out in the image below:

| Your Design|
|--------------|
|![image](https://user-images.githubusercontent.com/62938735/193421130-5cc9b99f-327b-4155-8b11-3ac0fa2f76a4.png)|